AnimView/frmSprites.cpp

```diff
@@ -20,91 +20,126 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-    pFilesGrid->Add(m_txtTable = new wxTextCtrl(this, wxID_ANY, L"X:\\ThemeHospital\\hospital\\QData\\Font00V.tab"), 1, wxALIGN_CENTER_VERTICAL | wxEXPAND);
-    pFilesGrid->Add(new wxButton(this, ID_BROWSE_TABLE, L"Browse..."), 0, wxALIGN_CENTER_VERTICAL);
-    pFilesGrid->Add(new wxStaticText(this, wxID_ANY, L"Data:"), 0, wxALIGN_CENTER_VERTICAL | wxALIGN_RIGHT);
-    pFilesGrid->Add(m_txtData = new wxTextCtrl(this, wxID_ANY, L""), 1, wxALIGN_CENTER_VERTICAL | wxEXPAND);
-    pFilesGrid->Add(new wxButton(this, ID_BROWSE_DATA, L"Browse..."), 0, wxALIGN_CENTER_VERTICAL);
-    pFilesGrid->Add(new wxStaticText(this, wxID_ANY, L"Palette:"), 0, wxALIGN_CENTER_VERTICAL | wxALIGN_RIGHT);
-    pFilesGrid->Add(m_txtPalette = new wxTextCtrl(this, wxID_ANY, L"X:\\ThemeHospital\\hospital\\Data\\MPalette.dat"), 1, wxALIGN_CENTER_VERTICAL | wxEXPAND);
-    pFilesGrid->Add(new wxButton(this, ID_BROWSE_PALETTE, L"Browse..."), 0, wxALIGN_CENTER_VERTICAL);
+    pFilesGrid->Add(
+            new wxStaticText(this, wxID_ANY, L"Table:"),
+            0,
+            wxALIGN_CENTER_VERTICAL | wxALIGN_RIGHT);
+    pFilesGrid->Add(
+            m_txtTable = new wxTextCtrl(
+                    this,
+                    wxID_ANY,
+                    L"X:\\ThemeHospital\\hospital\\QData\\Font00V.tab"),
+            1,
+            wxALIGN_CENTER_VERTICAL | wxEXPAND);
+    pFilesGrid->Add(
+            new wxButton(this, ID_BROWSE_TABLE, L"Browse..."),
+            0,
+            wxALIGN_CENTER_VERTICAL);
+    pFilesGrid->Add(
+            new wxStaticText(this, wxID_ANY, L"Data:"),
+            0,
+            wxALIGN_CENTER_VERTICAL | wxALIGN_RIGHT);
+    pFilesGrid->Add(
+            m_txtData = new wxTextCtrl(this, wxID_ANY, L""),
+            1,
+            wxALIGN_CENTER_VERTICAL | wxEXPAND);
+    pFilesGrid->Add(
+            new wxButton(this, ID_BROWSE_DATA, L"Browse..."),
+            0,
+            wxALIGN_CENTER_VERTICAL);
+    pFilesGrid->Add(
+            new wxStaticText(this, wxID_ANY, L"Palette:"),
+            0,
+            wxALIGN_CENTER_VERTICAL | wxALIGN_RIGHT);
+    pFilesGrid->Add(
+            m_txtPalette = new wxTextCtrl(
+                    this,
+                    wxID_ANY,
+                    L"X:\\ThemeHospital\\hospital\\Data\\MPalette.dat"),
+            1,
+            wxALIGN_CENTER_VERTICAL | wxEXPAND);
+    pFilesGrid->Add(
+            new wxButton(this, ID_BROWSE_PALETTE, L"Browse..."),
+            0,
+            wxALIGN_CENTER_VERTICAL);
     pFiles->Add(pFilesGrid, 0, wxEXPAND | wxALL, 1);
-    wxButton *pTmp;
-    pFiles->Add(pTmp = new wxButton(this, ID_LOAD, L"Load Simple"), 0, wxALIGN_CENTER | wxALL, 1);
-    pFiles->Add(pTmp = new wxButton(this, ID_LOAD_COMPLEX, L"Load Complex"), 0, wxALIGN_CENTER | wxALL, 1);
-    pFiles->Add(pTmp = new wxButton(this, ID_NEXT, L"Next"), 0, wxALIGN_CENTER | wxALL, 1);
+    wxButton* pTmp;
+    pFiles->Add(
+            pTmp = new wxButton(this, ID_LOAD, L"Load Simple"),
```

Comment:
Code is being really silly here, at least I don't see the need for `pTmp`.

Reply:
Out of scope - I got in a trap of refactoring everything earlier, but it can always be done later.